hbase-backup/src/test/java/org/apache/hadoop/hbase/backup/TestBackupDeleteWithCleanup.java

```diff
@@ -96,6 +119,70 @@ public void testBackupDeleteWithCleanupLogic() throws Exception {
 
     // Step 5: Verify System Table Update
     verifySystemTableUpdate(backupSystemTable, currentTime);
+
+    // Cleanup
+    deleteBackup(anotherBackupId);
+  }
+
+  @Test
+  public void testSingleBackupForceDelete() throws Exception {
+    // Step 1: Setup Backup Folders
+    long currentTime = EnvironmentEdgeManager.getDelegate().currentTime();
+    setupBackupFolders(currentTime);
+
+    // Log the directory structure before cleanup
+    logDirectoryStructure(fs, backupWalDir, "Directory structure BEFORE cleanup:");
+
+    // Step 2: Simulate Backup Creation
+    backupSystemTable.addContinuousBackupTableSet(Set.of(table1),
+      currentTime - (2 * ONE_DAY_IN_MILLISECONDS));
+
+    EnvironmentEdgeManager
+      .injectEdge(() -> System.currentTimeMillis() - (2 * ONE_DAY_IN_MILLISECONDS));
+
+    String backupId = fullTableBackup(Lists.newArrayList(table1));
+    assertTrue(checkSucceeded(backupId));
+
+    // Step 3: Run Delete Command
+    deleteBackup(backupId);
+
+    // Log the directory structure after cleanup
+    logDirectoryStructure(fs, backupWalDir, "Directory structure AFTER cleanup:");
+
+    // Step 4: Verify CONTINUOUS_BACKUP_REPLICATION_PEER is disabled
+    assertFalse("Backup replication peer should be disabled or removed",
+      continuousBackupReplicationPeerExists());
+
+    // Step 5: Verify that system table is updated to remove all the tables
+    Set<TableName> remainingTables = backupSystemTable.getContinuousBackupTableSet().keySet();
+    assertTrue("System table should have no tables after force delete", remainingTables.isEmpty());
```

Comment:
what is `force delete`? maybe change the wording to full backup is clear

```suggestion
    assertTrue("System table should have no tables after full backup is clear", remainingTables.isEmpty());
```